docs/rails.md

```diff
@@ -80,6 +84,12 @@ rebuild.)
 
     $ docker-compose build
 
+
+> *Note:* You'll notice the files created by `rails new` are owned
```

Comment:
Replace with the information I gave you above...as this is not in procedure but narrative style, tweak in this way

The files `django-admin` created are owned by root. This happens because the  container runs as the `root` user.  Change the ownership of the new files.

      `sudo chown -R $USER:$USER .`